src/ocm/services/StorageService.tsx

```diff
@@ -0,0 +1,38 @@
+import {
```

Comment:
I think this may be an issue with rebasing.

The file is not being used, and the `setOLMOperators` is being called from `HostDiscovery` which I think is correct.
Could it just be that it needs to be removed?

Reply:
Yes it should, thank you